joomla/joomla-cms
# [4.0] Require parameter in custom field Subfield

Pull Request for Issue # https://github.com/joomla/joomla-cms/issues/27736.

### Summary of Changes
I hid the require parameter in the Custom Field `subfield`.


### Testing Instructions
1. Create a custom field of the type `subfield`
2. See that the parameter require is shown. 
![Articles  New Field   test   Administration(1)](https://user-images.githubusercontent.com/9974686/73591747-e308ca80-44f2-11ea-9220-576c1dcef134.png)


3. Apply this patch and see, that the parameter require is not shown anymore.
![Articles  New Field   test   Administration](https://user-images.githubusercontent.com/9974686/73591721-768dcb80-44f2-11ea-8fd3-69429a9ee1aa.png)


### Expected result
If the option `required` is offered, the value of the field should have an impact. 
In the case of a `subfield`, technically there is always a value. Therefore, the assignment of this option does not matter. 

### Actual result

The field is shown unnecessarily here.

### Documentation Changes Required
Yes
https://docs.joomla.org/Custom_fields_type:_Subfields
https://docs.joomla.org/J3.x:Adding_custom_fields (Here the subfield is missing at all)
